core/tests/protractor_utils/LearnerDashboardPage.js

```diff
@@ -54,69 +54,47 @@ var LearnerDashboardPage = function() {
   };
 
   this.navigateToPlayLaterExplorationSection = async function() {
-    await waitFor.elementToBeClickable(
-      playLaterSection,
-      'Play Later Exploration Section tab takes too long to be clickable');
-    await playLaterSection.click();
+    await action.click('Play Later Explorations Tab', playLaterSection);
   };
 
   this.navigateToCompletedSection = async function() {
-    await waitFor.elementToBeClickable(
-      completedSection, 'Completed tab takes too long to be clickable');
-    await completedSection.click();
+    await action.click('Completed Tab', completedSection);
```

Comment:
I Just realized that we have to import `actions` module on top similar to waitFor else the `actions` will be undefined!

Reply:
Done